test/evergreen/run_format_configs.sh

```diff
@@ -21,20 +20,102 @@ fi
 
 success=0
 failure=0
+running=0
+parallel_jobs=8
+cleanup=0
+PID=""
+declare -a PID_LIST
+
+usage() {
+	echo "usage: $0 "
+	echo "    -j <num_of_jobs>  number of jobs to execute in parallel (defaults to 8)"
+
+	exit 1
+}
+
+# Wait for other format runs.
+wait_for_process()
+{
+	while true ; do
+		for process in ${PID_LIST[@]};do
+			ps $process > /dev/null
+			ps_exit_status=$?
+
+			if [ ${ps_exit_status} -eq "1" ] ; then
```

Comment:
Suggest combining them into a single line.

Reply:
Fixed in e86d6151fdda76661610cd6736e283617419375b